lib/IRGen/IRGenSIL.cpp

```diff
@@ -1938,59 +1972,129 @@ void IRGenSILFunction::visitGraphOperationInst(GraphOperationInst *i) {
 
   auto &silModule = CurSILFn->getModule();
 
+  auto *TFNewStatusFn = IGM.getTF_NewStatusFn();
+  auto status = Builder.CreateCall(TFNewStatusFn, {});
+
+  if (opInfo.getOperationName() != "Const") {
+    LLVM_DEBUG(llvm::dbgs()
+               << "Done with IRGen for dummy graph_op; setting explosion.\n");
+    // The added explosion is incorrect, but is good enough for unit testing.
+    Explosion e;
+    e.add(TFNewStatusFn);
+
+    SILValue result = i->getResults()[0];
+    setLoweredExplosion(result, e);
+    return;
+  }
+
   // The true return type is TFE_Context*, which is an opaque pointer, so it
   // maps to void* in the Swift-C calling convention. `eagerContext` has type
   // void*, or i8* in LLVM type system.
-  auto getContextSilFn = silModule.findFunction(
-      "_swift_tfc_GetGlobalEagerContext", SILLinkage::PublicExternal);
-  assert(getContextSilFn);
-  llvm::Constant *getContextFn =
-      IGM.getAddrOfSILFunction(getContextSilFn, NotForDefinition);
-  assert(getContextFn);
+  llvm::Function *getContextFn =
+      findFunction("_swift_tfc_GetGlobalEagerContext", silModule);
```

Comment:
Do you plan to move this one to runtimefunctions as well?

Reply:
How can I change those swift functions to be called as runtime functions?

If we can do that, we may not need https://github.com/apple/swift/pull/19556, which would be nice.

Reply:
Just make sure they have a convention(c) calling convention and public linkage and you can call them as C functions!

Reply:
Simple change:
Replace `@_silgen_name` with `@_cdecl` and you'll get C-convention functions.

Reply:
Thanks. Converted some entry points to `@_cdecl`, and added runtime function entries for them.

For the remaining ones, I'll convert them once we use the new AnyTensorHandle.